docs/install-upgrade.md

```diff
@@ -155,12 +155,47 @@ To install the MongoDB Community Kubernetes Operator:
 
 ## Upgrade the Operator
 
-To upgrade the MongoDB Community Kubernetes Operator to a specific version:
+The release v0.6.0 had some breaking changes (see https://github.com/mongodb/mongodb-kubernetes-operator/releases/tag/v0.6.0) requiring special steps to upgrade from a pre-0.6.0 Version.
+As always, have backups.
+Make sure you run commands in the correct namespace.
 
-1. Change the version in the Operator [resource definition](../config/manager/manager.yaml)
-2. Change to the directory in which you cloned the repository.
-3. Checkout the specific tag matching the operator version (e.g. `v0.5.1`)
-4. Invoke the following `kubectl` command to upgrade the [Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/).
-   ```
-   kubectl apply -f config/crd/bases/mongodbcommunity.mongodb.com_mongodbcommunity.yaml
-   ```
+1. Prepare for the upgrade.
+
+   a. Migrate your cr by updating apiVersion and kind to
+      ```
+      apiVersion: mongodbcommunity.mongodb.com/v1
+      kind: MongoDBCommunity
+      ```
+      If you upgrade from pre-0.3.0 you need to also add the field spec.users[n].scramCredentialsSecretName for each resource. This will be used to determine the name of the generated secret which stores MongoDB user credentials. This field must comply with DNS-1123 rules (must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character).
```

Comment:
Can you reference https://kubernetes.io/docs/concepts/overview/working-with-objects/names/ and maybe remove the parenthesis comment.

Reply:
Replaced the text in parenthesis with the reference link.